pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-holder.yaml

```diff
@@ -0,0 +1,63 @@
+# This DaemonSet is only active when the cluster is deployed with Multus enabled. Hopefully in the
+# future it will be part of every Ceph-CSI deployment. Its role is to "hold" the network mounts (for
+# CephFS) and maps (for RBD) that are created by the CSI driver. Essentially, it exposes its network
+# namespace into the Kubelet plugin directly and Ceph-CSI picks it up. When mounting or mapping, the
+# Ceph-CSI driver will use the network namespace of the DaemonSet instead of its own (the plugin
+# pod). This pod is not expected to be updated nor restarted unless the node reboots.
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-rbdplugin-holder
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: csi-rbdplugin-holder
+  updateStrategy:
+    # DO NOT CHANGE THE STRATEGY
+    # The pod is not supposed to be restarted on updates, it should never receive updates too.
+    # If it does then the connectivity to the Ceph cluster will be lost and the client I/O will
+    # fail.
+    type: OnDelete
```

Comment:
should we set `restartPolicy` to Never for this ds?

Reply:
If the container fails during its initial start I think it's good to try again. If it crashes for whatever reason restarting won't really change anything. How does that sound?

Reply:
Not sure its possible but yes its sounds good :+1: